src/main/java/uk/gov/pay/adminusers/exception/ServiceNotFoundException.java

```diff
@@ -1,6 +1,6 @@
 package uk.gov.pay.adminusers.exception;
 
-public class ServiceNotFoundException extends Throwable {
+public class ServiceNotFoundException extends NotFoundException {
```

Comment:
Making this match how we do it in direct-debit-connector by having the ServiceNotFoundException extend NotFoundException then just having an ExceptionMapper for the NotFoundException.

Reply:
Yeah, `extends Throwable` is just… _yeesh_.